scripts/generateAdvancedAWS.sh

```diff
@@ -0,0 +1,3 @@
+yarn build
+node ./scripts/generateAdvancedAWS.js
+cd aws-advanced-test
```

Comment:
I would not recommend to change the current working directory as this is quite implicit and not expected.
Some examples:
- Github clone will not cd in the cloned repo
- npm new will not
- rails new will not
- ...

In the GitHub Action workflow, we can use the [working-directory instruction](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory) to indicate to run the tests in the generated project:

```yml
      - name: Run Terraform format
        working-directory: ./aws-advanced-test
        run: terraform fmt -recursive -check

      - name: Run tfsec linter
        id: tfsec
        working-directory: ./aws-advanced-test
        uses: aquasecurity/tfsec-action@v1.0.3
        with:
          version: ${{ env.TFSEC_VERSION }}
```

Reply:
Good to know 👍, removed in [d261cf3](https://github.com/nimblehq/infrastructure-templates/pull/276/commits/d261cf39cbd5dadfc922ffbecabc8c2fc844ab95)